src/utils/__tests__/available-rpc.test.ts

```diff
@@ -9,13 +9,12 @@ chai.use(chaiAsPromised);
 const { expect } = chai;
 
 describe('available-rpc', () => {
-  it('Should check fallback provider cascade', async () => {
+  it.only('Should check fallback provider cascade', async () => {
```

Comment:
This is disabling all of the tests except this one. We need to have all the tests running.

If the whole test suite still works after removing `.only`, feel free to merge. :)

Reply:
Whoops